src/Microsoft.TestPlatform.ObjectModel/TestProcessAttachDebuggerPayload.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel
+{
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// The test process info payload.
+    /// </summary>
+    [DataContract]
+    public class TestProcessAttachDebuggerPayload
+    {
+        /// <summary>
+        /// Creates a new instance of this class.
+        /// </summary>
+        /// <param name="pid">The process id the debugger should attach to.</param>
+        public TestProcessAttachDebuggerPayload(int pid)
```

Comment:
>TestProcessAttachDebuggerPayload [](start = 15, length = 32)

Should we consider adding in a propertybag here to make this future proof?

Reply:
Thinking of additional debugger information.

---
In reply to: [423933703](https://github.com/microsoft/vstest/pull/2325#discussion_r423933703) [](ancestors = 423933703)

Reply:
Never mind, this is an attach.

---
In reply to: [423933966](https://github.com/microsoft/vstest/pull/2325#discussion_r423933966) [](ancestors = 423933966,423933703)